Windows/MainWindow.xaml.cs

```diff
@@ -95,11 +98,31 @@ public partial class MainWindow {
             }
         }
 
-        public bool SingleClickToEditMode { get; set; } = true;
+        public bool SingleClickToEditMode { get; set; } = false;
 
         public MainWindow() {
             var args = Environment.GetCommandLineArgs();
 
+            if (args.Length > 1)
+            {
+                foreach (var arg in args.Skip(1).ToArray())
+                {
+                    if (arg.Length == 0)
+                        return;
+
+                    string iPath = Path.GetDirectoryName(arg);
+                    string iFileName = Path.GetFileName(arg).Replace(".", "_");
+                    string oPath = Path.Combine(iPath, "_Json");
+                    string oFileName = iFileName + ".json";
+                    System.IO.Directory.CreateDirectory(oPath);
+                    string target = Path.Combine(oPath, oFileName);
+                    BatchLoad(arg);
+                    ExportJson(target);
+                }
+                System.Environment.Exit(0);
```

Comment:
This doesn't need to be exclusive. You could, for example, add a cmd flag for batch exporting and try to load every arg after it.

Reply:
Ahhh too much work.